APMrover2/RC_Channel.cpp

```diff
@@ -195,6 +207,10 @@ void RC_Channel_Rover::do_aux_function(const aux_func_t ch_option, const aux_swi
         rover.control_mode->handle_tack_request();
         break;
 
+    case AUX_FUNC::REVERSE_THROTTLE:
+        rover.reversed_throttle = (ch_flag == HIGH);
```

Comment:
This state should be in the RC class.  Possibly even the base class - Copter will get reverse throttle at some stage...